tests/src/test/scala/akka/kafka/scaladsl/CommittingSpec.scala

```diff
@@ -264,6 +264,118 @@ class CommittingSpec extends SpecBase with TestcontainersKafkaLike with Inside {
       control2.isShutdown.futureValue shouldBe Done
     }
 
+    "retry commits that occurred during rebalance" in assertAllStagesStopped {
```

Comment:
This test is quite complex.  See review summary comment.

Reply:
thank you, removed this test in favour of unit one
the only reason to add this test was removing of ```rebalanceInProgress``` flag, but I've realised that it had been redundant before my changes.